frontend/src/components/pages/Appointments.js

```diff
@@ -69,19 +140,46 @@ function Appointments() {
     );
   };
 
-  const UpcomingTab = () => {
-    // Example, feel free to remove or change
-    return null;
+  const Appointments = (props) => {
+    return (
+      <div>
+        <b className="appointment-tabs-title">{currentTab.title}</b>{" "}
+        <div className="appointments-background">
+          {props.data.map((appointmentsObject, index) => (
```

Comment:
(cont) and then just:
```suggestion
          {data.map((appointmentsObject, index) => (
```